web/source/kmwcallback.ts

```diff
@@ -73,48 +73,7 @@ if(!window['keyman']['initialized']) {
      * Description  Register and load the keyboard
      */    
     KeymanWeb['KR'] = kbdInterface['registerKeyboard'] = kbdInterface.registerKeyboard = function(Pk) {
-      // If initialization not yet complete, list the keyboard to be registered on completion of initialization
-      if(!keymanweb['initialized']) {
-        keymanweb.deferredKR.push(Pk); return;
-      }
-      
-      var Li,Lstub;
-
-      // Check if the active stub refers to this keyboard, else find applicable stub
-
-      var Ps=keymanweb._ActiveStub, savedActiveStub = keymanweb._ActiveStub;
-      if(!Ps || !('KI' in Ps) || (Ps['KI'] != Pk['KI'])) {         
-        // Find the first stub for this keyboard
-        for(Lstub=0;Lstub < keymanweb._KeyboardStubs.length; Lstub++) { // I1511 - array prototype extended
-          Ps=keymanweb._KeyboardStubs[Lstub];
-          if(Pk['KI'] == Ps['KI'])break;
-          Ps=null;
-        }
-      } 
-      
-      // Build 369: ensure active stub defined when loading local keyboards 
-      if(keymanweb._ActiveStub == null && Ps != null) {
-        keymanweb._ActiveStub = Ps;
-      }
-      
-      // Register the stub for this language (unless it is already registered)
-      // keymanweb.KRS(Ps?Ps:Pk); 
-
-      // Test if keyboard already loaded
-      for(Li=0; Li<keymanweb._Keyboards.length; Li++) {
-        if(Pk['KI'] == keymanweb._Keyboards[Li]['KI']) {
-          return;
-        }
-      }
-    
-      // Append to keyboards array
-      keymanweb._Keyboards=keymanweb._push(keymanweb._Keyboards,Pk);
-
-      // Execute any external (UI) code needed after loading keyboard
-      keymanweb.doKeyboardLoaded(Pk['KI']);
-
-      // Restore the originally-active stub to its prior state.  No need to change it permanently.
-      keymanweb._ActiveStub = savedActiveStub;
+      keymanweb.keyboardManager._RegisterKeyboard(Pk);
```

Comment:
Any reason why `_RegisterKeyboard` has `_TitleCase` name where many others don't in `keyboardManager`?